mods/cnc/maps/scb30ea/scb30ea-AI.lua

```diff
@@ -0,0 +1,321 @@
+--[[
+   Copyright 2007-2022 The OpenRA Developers (see AUTHORS)
```

Comment:
This line changed compared to the last released version too:
```suggestion
   Copyright (c) The OpenRA Developers and Contributors
```